client/src/pages/home/HomePage.js

```diff
@@ -1,19 +1,60 @@
-import React from "react";
+import React, { useState } from "react";
 import ProverbList from "./../proverb/ProverbList";
-
+import AddProverb from "./../proverb/AddProverb";
 import Section from "./../../components/Section";
-import Breadcrumb from './../../components/Breadcrumb'
+import Breadcrumb from "./../../components/Breadcrumb";
+import Modal from "./../../components/Modal";
+import Button from "./../../components/Button";
+
+import "./HomePage.css";
 
 const HomePage = () => {
+  //Modal States
+  const [modal, setModal] = useState({
+    isOpen: false,
+    type: "",
+  });
+
+  //Modal Handlers
+
+  const handleShowModal = (type) => async (e) => {
+    e.preventDefault();
+    if (type === "add") {
+      setModal({
+        isOpen: true,
+        type: "Add",
+      });
+    }
+  };
   return (
     <div>
-   <Section
+      <Modal
+        isOpen={modal.isOpen}
+        modalClose={() => setModal({ isOpen: false, type: undefined })}
```

Comment:
```suggestion
        modalClose={handleCloseModal}
```

you could create a function called handleCloseModal and use it as I wrote above. this could be an implementation:

```
function handleCloseModal() {
   setModal({ isOpen: false, type: undefined })
}
```
It's good to have named functions, they are easier to read and to debug than anonymous functions. I would advise
using arrow functions when they really improve the readability of the code or when you want to avoid context issues related to using the "this" object.

You can find a lot of interesting information about arrow functions here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions